result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+components:
+  schemas:
+    LldpNeighbors.State.Request:
+      description: >-
+        The request to retrieve LLDP neighbor information for a given interface.
+      type: object
+      properties:
+        lldp_interface_names:
+          description: >-
+            The names of LLDP interfaces for which neighbor information will be
+            retrieved. If no names are specified then the results will
+            contain neighbor information for all configured LLDP interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Lldp/properties/name"
+        neighbor_id_filters:
+          description: >-
+            Specify the neighbors for which information will be retuened. If the list is 
```

Comment:
retuened --> returned.
If the empty ... --> If empty...

Reply:
changed the description